docs/unified-test-documentation/dasharo-compatibility/30L-memtest-payload-support.md

```diff
@@ -0,0 +1,169 @@
+4. Select the key with a proper number for `Payload [memtest]`.
+5. Check if `Memtest86+` is available.
+
+**Expected result**
+
+The `Memtest86+` is visible at the top of the output.
+
+### MEM003.001 Memtest stability
+
+**Test description**
+
+This test aims to verify that the Memtest starts does not hang under DUT.
+
+**Test configuration data**
+
+1. `FIRMWARE` = coreboot
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the key with a proper number for `Payload [memtest]`.
+5. Check if `State: - Running...` is available.
+
+**Expected result**
+
+The `State: - Running...` is visible after a few seconds and confirms that the
+the test is in progress.
+
+### MEM004.001 Memtest refreshing by 'L' key
+
+**Test description**
+
+This test aims to verify that DUT refreshes Memtest properly.
+
+**Test configuration data**
+
+1. `FIRMWARE` = coreboot
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the key with a proper number for `BOOT_MENU_ENTRY`.
+5. Press the `L` key.
+6. Check if `L` refreshes output.
+
+**Expected result**
+
+The `Memtest86+` is visible before and after pressing `L`.
+
+### MEM005.001 Memtest refreshing by 'l' key
+
+**Test description**
+
+This test aims to verify that DUT refreshes Memtest properly.
+
+**Test configuration data**
+
+1. `FIRMWARE` = coreboot
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the key with a proper number for `BOOT_MENU_ENTRY`.
+5. Press the `l` key.
+6. Check if `l` refreshes output.
+
+**Expected result**
+
+The `Memtest86+` is visible before and after pressing `l`.
+
+### MEM006.001 Memtest completing
+
+**Test description**
+
+This test aims to verify that DUT completes Memtest.
+
+1. `FIRMWARE` = coreboot
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+2. Wait for boot until `BOOT_MENU_STRING` appears.
+3. Press `BOOT_MENU_KEY` to enter the boot menu.
+4. Select the key with a proper number for `Payload [memtest]`.
+5. Make sure that `State: - Running...` is available.
+6. Wait until `** Pass complete, no errors, press Esc to exit **` appears.
+
+**Expected result**
+
+After the test completes without any errors
+`** Pass complete, no errors, press Esc to exit **` message is visible on the
+bottom of the screen.
```

Comment:
please add a new line at the end of the file

Reply:
@damiankaras You keep suggesting that, why? I asked yesterday. I see no point here.

Reply:
So why has not the precommit push the new line in the eof?

Reply:
@damiankaras It does. There is no such issue here, so why it should do anything. We do not want to **literally add new empy line** at the end of file. It would be pointless and against the rules, in fact.

We need to add the `newline` character, though, if it is missing. Not a `new (empty) line`. Please read: https://stackoverflow.com/questions/16222530/why-do-i-need-vim-in-binary-mode-for-noeol-to-work

I guess you are confused between these.

Reply:
ok, I've read